src/client/Microsoft.Identity.Client/Internal/Requests/ClientCredentialRequest.cs

```diff
@@ -187,20 +202,78 @@ private async Task<AuthenticationResult> SendTokenRequestToAppTokenProviderAsync
             return authResult;
         }
 
+        /// <summary>
+        /// Checks if the token should be used from the cache.
```

Comment:
```suggestion
        /// Checks if the token should be used from the cache and returns the cached access token if applicable.
```

Reply:
This change is not applied and marked resolved. Can you share the reason why?

Reply:
good find @neha-bhargava , applied now.